### ISSUE TITLE
too much robin warning

### ISSUE THREAD

**Issue by Hokila** [outboundio/iOS#3]
I install your library to my app.
and get many warning like this

warning: /Users/robin/Library/Developer/Xcode/DerivedData/ModuleCache/3P8QWV00DNLL6/Foundation-FNRSQMLKYYBK.pcm: No such file or directory
warning: /Users/robin/Library/Developer/Xcode/DerivedData/ModuleCache/3P8QWV00DNLL6/UIKit-3P5M9I5R2LQCD.pcm: No such file or directory

seems you put absolute path when building library.
Please fix these warnings ASAP

**Comment by Tang8330** [comment]
Thanks Hokila!
Looking into it right now :) Stay tuned.

**Comment by Tang8330** [comment]
Hi Hokila,
I just ran our sample app with the Outbound SDK implementation, and was not able to reproduce this warning. Could you tell me how you got this warning?

Also, have you tried to clean and rebuild your project to see if that represses the issue.

Best, Robin

**Comment by Hokila** [comment]
Hi Robin,

I get warnings when app build.
http://cl.ly/2A05073z2X3v

I think it's because you build library and ran sample app in same computer.
so you can not these warnings.

BRs
Hokila

**Comment by Tang8330** [comment]
Hello!

Is this the repo you are trying to build it on? https://github.com/Hokila/OutboundTest

I have cloned that repo + created a new sample app and still not getting that warning. (Have done both of these on another machine without /Users/robin

Robin

**Comment by Hokila** [comment]
Hi Robin,
Yes ,I use this repo to test integration.
It was for demo server not reachable problem with SDK version 1.0

just minutes ago I update to 1.0.1 
I think it's able to show warnings now

BRs
HOkila

**Comment by Tang8330** [comment]
Hi Hokila

I have updated on your repo on a separate computer and ran the app.
This is all I am getting https://gist.github.com/anonymous/588c98dec432cbfb4bfa16290abbc57b

Further more investigation on the internet suggests that:
```
What solves this for me is to create a new scheme. 
Click on Product-> Scheme -> New Scheme and just hit OK.
Then my app seems to build and run.
```
Could you try that? On the SDK, I have changed the build path to be relative instead of custom. All that has happened from it is that /DerivedData/ folder has been created. (Which are just the build files that get generated when compiling the app)

Thanks for your help!
Robin

**Comment by Hokila** [comment]
Hi Robin,

Seems you see these warnings.
Temporarily  I change Build Options -> Debug Information Format ,
from "DWARF with dSYM file " to "DWARF"
so warnings will not showup, even it's still exist.

Wait you release new library.I will update to new version then.

BRs
Hokila

**Comment by Tang8330** [comment]
Hi Hokila!

Just to clarify - I cloned your repo, and was not able to see any of these warnings.
On the SDK side, I tried changing this to relative path does not actually make any config changes. So pushing out a new library would not solve this.

However, moving forward you mentioned you changed your Build Options to DWARF. 
Does this solution solve your problem? If not, we can arrange to hop on a call? My email is (robin@outbound.io)

Best,
Robin
